packages/js/tracking-jsdoc/README.md

```diff
@@ -0,0 +1,83 @@
+# woo-tracking-jsdoc
+
+JSDoc template to report Tracking events to markdown file.
+
+## Usage
+
+1. Install or link the package via npm
+2. Add your `/TRACKING.md` template
+
+   ```md
+   # Usage Tracking
+   
+   Some nice general description.
+   
+   <woo-tracking-jsdoc></woo-tracking-jsdoc>
+   ```
+3. Generate the docs
+   ```sh
+   jsdoc -r your/source/files/ -t ./node_modules/woo-tracking-jsdoc
+   ```
+
+
+
+## Config
+
+You may add any of the following properties to your JSDoc config (`.jsdocrc.json`) to change those default values:
+```js
+{
+  "templates": {
+    "woo-tracking-jsdoc": {
```

Comment:
:grey_question: On one hand, I'd make it consistent with the package name. On the other, it's pretty lengthy :|
Maybe we could consider renaming the package to "woo-"?

Reply:
I'd name all refs to `woocommerce-grow-{package}`

Being `{package}` the folder name (tracking-jsdoc)

Same in `package.json` file (about the name of the repo) 

` "name": "woocommerce-grow-tracking-jsdoc",`

Reply:
I'd name all refs to `woocommerce-grow-{package}`

Being `{package}` the folder name (tracking-jsdoc)

Same in `package.json` file (about the name of the repo) 

` "name": "woocommerce-grow-tracking-jsdoc",`

Reply:
I definitely take the point of consistency.
But it makes [a consumer's config](https://github.com/woocommerce/google-listings-and-ads/pull/1521/files#diff-58b92f5053951edae5cd71a5d73af0a8b51924f261aeb87b0d8cf8e803ce23cbL1-L20) longer and a bit blurred.

:nail_care: :scroll: WDYT of changing the convention to `woo-grow-{package}` (here and for others, probably as a new PR )
```json
{
  "opts": {
    "recurse": true,
    "template": "woocommerce-grow-tracking-jsdoc"
  },
  "plugins": [
    "woocommerce-grow-tracking-jsdoc/tilde-alias",
    "woocommerce-grow-tracking-jsdoc/fires-description",
    "jsdoc-plugin-typescript",
    "jsdoc-advanced-types-plugin",
    "jsdoc-plugin-intersection"
  ],
  "typescript": {
    "moduleRoot": "js/src"
  },
  "templates": {
    "woocommerce-grow-tracking-jsdoc": {
      "path": "src/Tracking/README.md"
    }
  }
}
```
vs
```json
{
  "opts": {
    "recurse": true,
    "template": "woo-grow-tracking-jsdoc"
  },
  "plugins": [
    "woo-grow-tracking-jsdoc/tilde-alias",
    "woo-grow-tracking-jsdoc/fires-description",
    "jsdoc-plugin-typescript",
    "jsdoc-advanced-types-plugin",
    "jsdoc-plugin-intersection"
  ],
  "typescript": {
    "moduleRoot": "js/src"
  },
  "templates": {
    "woo-grow-tracking-jsdoc": {
      "path": "src/Tracking/README.md"
    }
  }
}
```

Reply:
I don't have any preference for naming 👌

Reply:
I'll change it here to `woocommerce-grow-tracking-jsdoc` everywhere, to keep it consistent. Then I'll create an issue to discuss and potentially change it everywhere.